src/TopBar/index.tsx

```diff
@@ -15,16 +15,28 @@ export interface Props {
   title?: React.ReactNode;
   customLogo?: React.ReactNode;
   onLogoClick?: Function;
+  theme?: TopBarTheme;
+}
+
+export enum TopBarTheme {
+  DEFAULT = "default",
+  PLAIN = "plain",
 }
 
 /**
  * Global page-level header component.
  */
 export class TopBar extends React.PureComponent<Props> {
+  static defaultProps: Partial<Props> = {
+    theme: TopBarTheme.PLAIN,
+  };
+
   static Button = TopBarButton;
 
+  static Theme = TopBarTheme;
+
   render() {
-    const { children, className, title, customLogo } = this.props;
+    const { children, className, title, customLogo, theme } = this.props;
```

Comment:
Question specific to the FamPo usage: One other difference between the current TopBar and the FamPo TopBar is the existence of side gutters (that vary based on screen width).

![fampo](https://user-images.githubusercontent.com/12616928/77091968-85f3a600-69c6-11ea-92e2-0a93ad58bad2.png)

Any thoughts on how we should tackle that? We could build hardcoded gutters into the component, though we'd have to keep those in sync with the gutters we set in the FamPo. Alternatively, we could expose some `className` style prop for this.

Reply:
yeah, was wondering about a good way to handle that and open to suggestions! settled on letting FamPo adding the padding - just throw the top bar in a wrapper div and pad that with the same padding as the content pane

sounding like we'll have better standardised spacing on the grid in dewey2, so held off on additions here for now

Reply:
Makes perfect sense!